app/routing/Loadable/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactLoadable from 'react-loadable';
+import PropTypes from 'prop-types';
+
+import { getAsyncInjectors } from 'utils/asyncInjectors';
+
+const Loadable = ({ loader, LoadingComponent: CustomLoadingComponent, ...rest }) =>
+  class InnerLoadable extends React.Component {
+    static contextTypes = {
+      store: PropTypes.object,
+      defaultLoadingComponent: PropTypes.any,
+    };
+
+    loaderWithAsyncInjectors = () => {
+      if (loader) {
+        return loader(getAsyncInjectors(this.context.store))
+          .then((component) => component.default ? component.default : component);
+      }
+      return Promise.resolve(null);
+    };
+
+    emptyLoadingComponent = () => null;
+
+    loadableComponent = ReactLoadable({
+      ...rest,
+      loader: this.loaderWithAsyncInjectors,
+      LoadingComponent: CustomLoadingComponent || this.context.defaultLoadingComponent || this.emptyLoadingComponent,
```

Comment:
By the way, I tried running npm-check-updates on this branch and noticed react-loadable 4 is out, and this parameter was renamed to `loading`. Should we use the latest version?